### ISSUE TITLE
High DNS queries

### ISSUE THREAD

**Issue by krisaggio** [skuethe/MMM-Spotify#68]
Hi There,

I just notice that this module on idle state keeps pull up to 6 queries per second DNS resolution for api.spotify.com, while in a play state is one query every 5 seconds. Is it possible to reduce the queries on the idle state? 

ACTIVE:

```bash
9:59:35.722718 IP 172.17.0.1.56514 > pi.hole.domain: 43341+ A? api.spotify.com. (33)
19:59:35.722859 IP 172.17.0.1.56514 > pi.hole.domain: 54352+ AAAA? api.spotify.com. (33)
19:59:40.891329 IP 172.17.0.1.39103 > pi.hole.domain: 38366+ A? api.spotify.com. (33)
19:59:40.891441 IP 172.17.0.1.39103 > pi.hole.domain: 14714+ AAAA? api.spotify.com. (33)
19:59:46.015792 IP 172.17.0.1.43101 > pi.hole.domain: 32471+ A? api.spotify.com. (33)
19:59:46.017148 IP 172.17.0.1.43101 > pi.hole.domain: 42495+ AAAA? api.spotify.com. (33)
19:59:51.459680 IP 172.17.0.1.41866 > pi.hole.domain: 38071+ A? api.spotify.com. (33)
19:59:51.459807 IP 172.17.0.1.41866 > pi.hole.domain: 52685+ AAAA? api.spotify.com. (33)
19:59:56.636832 IP 172.17.0.1.40110 > pi.hole.domain: 39829+ A? api.spotify.com. (33)
19:59:56.637025 IP 172.17.0.1.40110 > pi.hole.domain: 30526+ AAAA? api.spotify.com. (33)
20:00:01.763523 IP 172.17.0.1.46347 > pi.hole.domain: 65004+ A? api.spotify.com. (33)
20:00:01.763835 IP 172.17.0.1.46347 > pi.hole.domain: 29810+ AAAA? api.spotify.com. (33)
20:00:06.958511 IP 172.17.0.1.60467 > pi.hole.domain: 36635+ A? api.spotify.com. (33)
20:00:06.958654 IP 172.17.0.1.60467 > pi.hole.domain: 49619+ AAAA? api.spotify.com. (33)
```
IDLE:

```bash
20:00:12.228973 IP 172.17.0.1.44411 > pi.hole.domain: 47810+ A? api.spotify.com. (33)
20:00:12.229069 IP 172.17.0.1.44411 > pi.hole.domain: 2579+ AAAA? api.spotify.com. (33)
20:00:17.385357 IP 172.17.0.1.41629 > pi.hole.domain: 2499+ A? api.spotify.com. (33)
20:00:17.385557 IP 172.17.0.1.41629 > pi.hole.domain: 24242+ AAAA? api.spotify.com. (33)
20:00:17.509869 IP 172.17.0.1.57280 > pi.hole.domain: 38450+ A? api.spotify.com. (33)
20:00:17.510130 IP 172.17.0.1.57280 > pi.hole.domain: 49190+ AAAA? api.spotify.com. (33)
20:00:17.716247 IP 172.17.0.1.49279 > pi.hole.domain: 3056+ A? api.spotify.com. (33)
20:00:17.716383 IP 172.17.0.1.49279 > pi.hole.domain: 6093+ AAAA? api.spotify.com. (33)
20:00:17.952301 IP 172.17.0.1.39722 > pi.hole.domain: 61569+ A? api.spotify.com. (33)
20:00:17.952468 IP 172.17.0.1.39722 > pi.hole.domain: 7734+ AAAA? api.spotify.com. (33)
```

Amazing module BTW!!! 

Cheers!

**Comment by bugsounet** [comment]
hi,

Thank for this report,
I confirm, it's a bug. It's a loop without timer ...
I have fixed it with v1.3.1

update / retry and tell me if ok

**Comment by krisaggio** [comment]
Hi Cedric,

Brilliant, that worked!

Thanks for such a quick fix  ;-)
